components/IscnUploadedInfo.vue

```diff
@@ -6,22 +6,24 @@
     <!-- header -->
     <IscnFormHeader :step="step" :total-step="4" />
     <!-- guide title -->
-    <div class="flex items-center justify-between mt-[12px] mb-[28px]">
+    <div class="flex items-center justify-start mt-[12px]">
       <Label
         align="middle"
         :text="$t('IscnUploaded.guide.title')"
         class="text-medium-gray"
       />
-      <Button
-        preset="plain"
-        :text="$t('NFTPortal.button.download.iscn')"
-        @click="handleClickDownload"
-      >
-        <template #prepend>
-          <IconDownload />
-        </template>
-      </Button>
     </div>
+    <IscnEditBar
+      class="my-[24px]"
+      :is-iscn-owner="true"
+      :iscn-id="iscnId"
+      :is-show-mint-button="true"
+      :is-nft-book="type === 'Book'"
+      :should-show-mint-button="true"
```

Comment:
What is this prop and how is it different from is show mint button above?

Reply:
[🎨 Remove duplicate props](https://github.com/likecoin/app-like-co/pull/438/commits/33a09c1376ae651a7935ea8cea2d08f4cfe36fe7)